DB/NEW_KT_DB/DataAccess/ObjectManager.py

```diff
@@ -12,34 +12,56 @@ def __init__(self, db_file: str):
     # for internal use only:
 
     # Riki7649255 based on rachel-8511
-    def create_management_table(self, table_name, table_structure='object_id INTEGER PRIMARY KEY AUTOINCREMENT,type_object TEXT NOT NULL,metadata TEXT NOT NULL')
+    def create_management_table(self, table_name, table_structure='object_id INTEGER PRIMARY KEY AUTOINCREMENT,type_object TEXT NOT NULL,metadata TEXT NOT NULL'):
+        self.db_manager.create_table(table_name, table_structure)
+    
+    def create_management_table_with_str_id(self, table_name, table_structure='object_id TEXT NOT NULL PRIMARY KEY ,metadata TEXT NOT NULL'):
         self.db_manager.create_table(table_name, table_structure)
 
     
     # Riki7649255 based on saraNoigershel
-    def insert_object_to_management_table(self, table_name, object):
-        self.db_manager.insert_data_into_table(table_name, object)
+    def insert_object_to_management_table(self, table_name, object_metadata):
```

Comment:
please specify what metadata should be sent to the function

Reply:
The metadata json each object have

Reply:
is this equivalent to `object.to_dict()`?